### DISCUSSION TITLE
For importing the entire planet, would a juiced up server beyond recommended specs speed it up?

### DISCUSSION THREAD

**Discussion by meyerovb** [mediagis/nominatim-docker#428] [Q&A] [answered]
Would using a 128gb ram 32 core cpu make it substantially faster for the initial load than 64gb ram 16 cores?

**Comment by mtmail** [comment, marked as answer]
All CPU cores will likely be 100% utilized for more than 48 hours. By adding more CPU cores you will save hours, possibly dozens of hours. Extra RAM helps, too.

**Reply by meyerovb** [reply]
What about shm_size, at what size does that start giving diminished returns?

**Reply by svupper** [reply]
shm_size = 16gb should be good
